index.html

```diff
@@ -5,35 +5,48 @@
         <link href="reset.css" rel="stylesheet">
     </head>
     <body>
-        <main>
-            <nav>
-                <ul class="nav-list1">
-                    <li>News</li>
-                    <li>Events</li>
-                    <li>Business</li>
-                    <li>AMAs</li>
-                    <li>Spaces</li>
-                    <li>Terms &#38; Conditions</li>
+        <nav>
+            <ul class="nav-list1">
+                 <li>News</li>
+                <li>Events</li>
+                <li>Business</li>
+                <li>AMAs</li>
+                <li>Spaces</li>
+                <li>Terms &#38; Conditions</li>
+            </ul>
+            <div class="nav-list2">
+                <div class="nl-2-1">
+                <ul>
+                    <li>LASTEST</li>
+                    <li>HARD FORK</li>
+                    <li>PLUGGED</li>
+                    <li>FUNDAMENTALS</li>
+                    <li>WORK 2030</li>
                 </ul>
-                <div class="nav-list2">
```

Comment:
For this section around **503px** and below there is this issue 
![resize](https://user-images.githubusercontent.com/38801423/62339070-c5599a80-b4a8-11e9-9b75-d735d4534bcd.png)